src/Spryker/Zed/EventBehavior/Business/Model/TriggerManager.php

```diff
@@ -40,9 +42,9 @@ class TriggerManager implements TriggerManagerInterface
     protected $config;
 
     /**
-     * @var bool|null
+     * @var bool
      */
-    protected static $eventBehaviorTableExists;
+    protected static $eventBehaviorTableExists = true;
```

Comment:
I prefer to have null as default to indicate that it's not checked. But this is not a blocker for me.